lib/rules/no-unused-expressions.js

```diff
@@ -55,17 +58,34 @@ module.exports = function(context) {
                 directives(parent).indexOf(node) >= 0;
     }
 
-    return {
-        "ExpressionStatement": function(node) {
+    /**
+     * @param {ASTNode} node - any node
+     * @returns {Boolean} whether the given node is a valid expression
```

Comment:
Would you like me to lower-case all of them in the file while I'm at it (another case of me adhering to consistency)?